content/blog/2016/2016-06-16-parallel-test-executor-plugin.adoc

```diff
@@ -68,6 +68,14 @@ void runTests(def args) {
 void mvn(def args) { /* ... */ }
 ----
 
+[NOTE]
+====
+This pipeline expects to be run as part of a multibranch pipeline.
+To run it stand-alone, replace the `checkout scm` step with
```

Comment:
Oh sorry, one more thing I didn't fully realize earlier. `checkout scm` works properly in multibranch pipelines OR stand-alone ones with "Pipeline from SCM". You, technically, only need `git` if you're entering in Pipeline script directly into the Jenkins configuration